test/widgets/profile/edit_brith_date_page_test.dart

```diff
@@ -0,0 +1,140 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:motis_mitfahr_app/account/models/profile.dart';
+import 'package:motis_mitfahr_app/account/pages/edit_account/edit_birth_date_page.dart';
+import 'package:motis_mitfahr_app/account/pages/profile_page.dart';
+import 'package:motis_mitfahr_app/util/locale_manager.dart';
+import 'package:motis_mitfahr_app/util/supabase.dart';
+import 'package:supabase_flutter/supabase_flutter.dart';
+
+import '../../util/factories/profile_factory.dart';
+import '../../util/mocks/mock_server.dart';
+import '../../util/mocks/request_processor.dart';
+import '../../util/mocks/request_processor.mocks.dart';
+import '../../util/pump_material.dart';
+
+void main() {
+  late Profile profile;
+  final MockRequestProcessor processor = MockRequestProcessor();
+  const String email = 'motismitfahrapp@gmail.com';
+  const String authId = '123';
+
+  setUpAll(() async {
+    MockServer.setProcessor(processor);
+  });
+
+  setUp(() {
+    profile = ProfileFactory().generateFake(id: 1);
+    SupabaseManager.setCurrentProfile(profile);
+    whenRequest(
+      processor,
+      urlMatcher: startsWith('/auth/v1/token'),
+      bodyMatcher: containsPair('email', 'motismitfahrapp@gmail.com'),
+      methodMatcher: equals('POST'),
+    ).thenReturnJson({
+      'access_token': authId,
+      'token_type': 'bearer',
+      'user': User(
+          id: authId,
+          appMetadata: {},
+          userMetadata: {},
+          aud: 'public',
+          createdAt: DateTime.now().toIso8601String(),
+          email: email),
+      'email': email,
+    });
+    whenRequest(processor, urlMatcher: contains('/rest/v1/profiles')).thenReturnJson(profile.toJsonForApi());
+  });
+  group('edit_birth_date_page', () {
+    testWidgets('birthDate TextField', (WidgetTester tester) async {
+      //load page
+      await pumpMaterial(tester, EditBirthDatePage(profile));
+      await tester.pump();
+
+      //check if birthDate is displayed
+      expect(find.text(localeManager.formatDate(profile.birthDate!)), findsOneWidget);
+
+      //check if birthDate TextField is displayed
+      final Finder birthDateInput = find.byKey(const Key('birthDateInput'));
+      expect(birthDateInput, findsOneWidget);
+
+      //check if birthDate TextField is editable
+      await tester.tap(birthDateInput);
+      await tester.pumpAndSettle();
+      expect(find.byType(DatePickerDialog), findsOneWidget);
+    });
+    testWidgets('change birthDate', (WidgetTester tester) async {
+      //load page
+      await pumpMaterial(tester, EditBirthDatePage(profile));
+      await tester.pump();
+
+      //check if birthDate is displayed
```

Comment:
didn't you do exactly this in line 54 alraedy? these tests have a little too much overlap